engine/render/src/render/program_utils.cpp

```diff
@@ -64,6 +73,8 @@ namespace dmRender
         dmVMath::Vector4* default_values = 0;
         uint32_t sampler_index = 0;
 
+        bool program_language_glsl = IsContextLanguageGlsl(dmGraphics::GetShaderProgramLanguage(graphics_context));
```

Comment:
Will there every be a case where we have two different language types at the same time? E.g. will compute always be the same as the vertex/fragment shaders? And same versions?

E.g. rather than have a duplicate function with an extra context, I think it might be better to store the language in the HProgram instead?

Reply:
Yes, that's probably a better idea.. Right now we only support a single shader language per context, but we need to fix that for other types of shaders (i.e compute), and especially since we can specify the version directly in a shader.